content/docs/command-reference/dag.md

```diff
@@ -21,19 +21,19 @@ show the full project DAG.
 ### Directed acyclic graph
 
 A data pipeline, in general, is a series of data processing
-[stages](/doc/command-reference/run) (for example, console commands that take an
-input and produce an outcome). The connections between stages are formed by the
-<abbr>output</abbr> of one turning into the <abbr>dependency</abbr> of another.
-A pipeline may produce intermediate data, and has a final result.
+[stages](/doc/command-reference/stage) (for example, console commands that take
+an input and produce an outcome). The connections between stages are formed by
+the <abbr>output</abbr> of one turning into the <abbr>dependency</abbr> of
+another. A pipeline may produce intermediate data, and has a final result.
 
 Data science and machine learning pipelines typically start with large raw
 datasets, include intermediate featurization and training stages, and produce a
 final model, as well as accuracy [metrics](/doc/command-reference/metrics).
 
 In DVC, pipeline stages and commands, their data I/O, interdependencies, and
 results (intermediate or final) are specified in `dvc.yaml`, which can be
-written manually or built using the helper command `dvc run`. This allows DVC to
-restore one or more pipelines later (see `dvc repro`).
+written manually or built using the helper command `dvc stage add`. This allows
+DVC to restore one or more pipelines later (see `dvc exp run` and `dvc repro`).
```

Comment:
Hmmmmm not sure we need to insert the `exp` reference here 🤔

Reply:
I guess it doesn't hurt. OK I'll solve the conflicts and merge this instead ⌛

Reply:
p.s. we should probably check that we're mentioning `dvc dag` from `exp` references too?

Reply:
OK. Extracted this into https://github.com/iterative/dvc.org/issues/3230 for now.